### PR TITLE
Update anonoymisation section in rdm-personal.md

### LINKED ISSUE #2490: Add additional resources to 'anonymisation' subheading of personal data management
# A Beginner's Guide to Contributing to _The Turing Way_

This is a guide for getting started with contributing to The Turing Way during the Collaborations Workshop in October 2022. We're so glad to have you here! 🎉 

Below you'll find some instructions for contributing to the project. If you have any questions, please let @BrainonSilicon or @EKaroune know on Zoom so that we can assist you.

## Summary
In the [Personal data management chapter](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-personal.html), in the anonymisation subheading, another resource can be added.

## Guide to Contribution 

1. Comment on the issue with the people working on the issue using their GitHub usernames.

2. Visit the relevant part of the repository to address this issue: https://github.com/alan-turing-institute/the-turing-way/tree/main/book/website/reproducible-research/rdm

3. Navigate to the 'anonymisation' subheading and search for this statement:
> "For more information about anonymisation you can watch [this webinar by Enrico Glerean](https://www.youtube.com/watch?v=ILXeA4fx3cI)."

4. Click 'Edit this file' to correct the wording.

Suggested Correction:
> "For more information about anonymisation you can watch[ this webinar by Enrico Glerean](https://www.youtube.com/watch?v=ILXeA4fx3cI), a webinar on [Amnesia - a tool to make anonymisation easy](https://www.youtube.com/watch?v=9wu_xGeYsQw) or [read an explanation by the finnish social science data archive](https://www.fsd.tuni.fi/en/services/data-management-guidelines/anonymisation-and-identifiers/)."

5. Use Markdown format to correct the wording on the page:
`For more information about anonymisation you can watch[ this webinar by Enrico Glerean](https://www.youtube.com/watch?v=ILXeA4fx3cI), a webinar on [Amnesia - a tool to make anonymisation easy](https://www.youtube.com/watch?v=9wu_xGeYsQw) or [read an explanation by the finnish social science data archive](https://www.fsd.tuni.fi/en/services/data-management-guidelines/anonymisation-and-identifiers/)`

6. Click 'Preview' to check that the changes look the way you'd like them too!

7. Navigate to 'Propose changes' at the bottom of the page: edit the title and the description of the issue to describe your update to the community.

8. Click 'Propose changes'. You will be redirected to a new page to make a Pull Request within the project. 

9. There will be template text in the Pull Request as is: don't worry too much about that! Erase the text, then link the issue tag using a`#`. You can find the issue number next to the title of this issue (this one of #2490).

Example: `this pull request fixes #2490`

10. Click 'Create Pull Request'! Congratulations! 🎉 

11. A reviewer will review your pull request, and ask you if you need to make any changes before it can be approved.

12. There are a number of checks your project will undergo: to check for merge conflicts, inclusive language, and formatting issues. Wait until those checks are complete before merging.

13. Once your pull request is approved and the checks are complete, you can merge your change into the repository! ✅

### PR DESCRIPTION
**Pull request by anicusan**
Added multiple materials on anonymisation

### Summary

Fixes #2490 

### List of changes proposed in this PR (pull-request)

* Added more materials to the anonymisation section, as instructed by the issue

### What should a reviewer concentrate their feedback on?

- Please check the resources added.

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: @Elisarau @anicusan @annarae13 @Zeena-Shawa

### CODE REVIEW

File: book/website/reproducible-research/rdm/rdm-personal.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ NOTE: In this instance, the linkage file should be encrypted and stored securely
...
 Further, data and outputs containing small cell counts may be potentially disclosive, particularly where samples are drawn from small populations or include cases with extreme values or relatively rare characteristics.
    * As such, when intending to share potentially identifiable data or the outputs generated from the data, you may need to consider more advanced anonymisation techniques such as statistical disclosure control (SDC, see [this handbook](https://securedatagroup.org/sdc-handbook/) for more information).
-* For more information about anonymisation you can watch [this webinar by Enrico Glerean](https://www.youtube.com/watch?v=ILXeA4fx3cI).
+* For more information about anonymisation you can watch[ this webinar by Enrico Glerean](https://www.youtube.com/watch?v=ILXeA4fx3cI), a webinar on [Amnesia - a tool to make anonymisation easy](https://www.youtube.com/watch?v=9wu_xGeYsQw) or [read an explanation by the finnish social science data archive](https://www.fsd.tuni.fi/en/services/data-management-guidelines/anonymisation-and-identifiers/)
```
Comment (da5nsy) [inline book/website/reproducible-research/rdm/rdm-personal.md:133]:
```suggestion
* For more information about anonymisation you can watch [this webinar by Enrico Glerean](https://www.youtube.com/watch?v=ILXeA4fx3cI), a webinar on [Amnesia - a tool to make anonymisation easy](https://www.youtube.com/watch?v=9wu_xGeYsQw), or read an [explanation by the Finnish social science data archive](https://www.fsd.tuni.fi/en/services/data-management-guidelines/anonymisation-and-identifiers/).
```

**Review by da5nsy — APPROVED** [review]

**Comment by da5nsy** [comment]
@all-contributors please add @Elisarau, @anicusan, @annarae13, and @Zeena-Shawa for content

**Comment by da5nsy** [comment]
@all-contributors please add @Elisarau for content

**Comment by da5nsy** [comment]
@all-contributors please add @anicusan for content

**Comment by da5nsy** [comment]
@all-contributors please add @annarae13 for content

**Comment by da5nsy** [comment]
@all-contributors please add @Zeena-Shawa for content

**Review by EstherPlomp — APPROVED** [review]
Looks fantastic, thanks for adding!! Everything looks good, so please feel free to merge it :)

**Pull request merged**